package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui-kit",
-  "version": "0.11.72",
+  "version": "0.11.73",
```

Comment:
Don't bump versions until you are releasing. What if a different PR gets merged and updates the version before yours?